.ci_support/linux_64_c_compiler_version11cuda_compilernvcccuda_compiler_version11.8cxx_compiler_version11fortran_compiler_version11mpi_typeconda.yaml

```diff
@@ -1,29 +1,27 @@
 c_compiler:
 - gcc
 c_compiler_version:
-- '12'
+- '11'
 cdt_name:
-- cos6
+- cos7
```

Comment:
The CUDA 11.8 Docker image [is on CentOS 7]( https://github.com/conda-forge/docker-images/blob/6d30f8fc76d5542a15330112f5b705bcc78b1887/.github/workflows/ci.yaml#L139-L143 ). So this got bumped from CentOS 6 to 7

Also note that `linux_aarch64` and `linux_ppc64le` already are using CentOS 7. So this really only affects `linux_64` (bringing it inline with the others)

Am guessing the long tail of CentOS 6 is diminishing. Though wouldn't be surprised if there are still some CentOS 6 users out there

Anyways just mentioning it for awareness